ceph.spec.in

```diff
@@ -33,10 +33,12 @@
 %bcond_with selinux
 %endif
 
-# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ and openSUSE TW
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ (not openSUSE)
 %if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} >= 1315
+%if ! 0%{?is_openSUSE}
```

Comment:
Actually, since ceph is only built for SLE 12, there's little point in checking explicitly for 1315. The following would be sufficient:

```
%if 0%{?fedora} || 0%{?rhel} >= 6 || ( 0%{?suse_version} && ! 0%{?is_openSUSE} )
```